src/Controller/Backend/FileManager.php

```diff
@@ -359,26 +359,28 @@ private function getRelatedFiles(FileInterface $file)
 
     private function getCodeMirrorPlugins(FileInterface $file)
     {
+        $all = ['fold/foldcode', 'fold/foldgutter', 'fold/indent-fold'];
+
         switch (strtolower($file->getExtension())) {
             case 'twig':
             case 'html':
             case 'htm':
-                return ['xml', 'javascript', 'css', 'htmlmixed'];
+                return array_merge($all, ['overlay', 'htmltwig', 'xml', 'javascript', 'css', 'htmlmixed', 'twig']);
             case 'php':
-                return ['matchbrackets', 'javascript', 'css', 'htmlmixed', 'clike', 'php'];
+                return array_merge($all, ['matchbrackets', 'javascript', 'css', 'htmlmixed', 'clike', 'php']);
             case 'yml':
             case 'yaml':
-                return ['yaml'];
+                return array_merge($all, ['yaml']);
             case 'md':
             case 'markdown':
-                return ['markdown'];
+                return array_merge($all, ['markdown']);
             case 'css':
             case 'less':
-                return ['css'];
+                return array_merge($all, ['css']);
             case 'js':
-                return ['javascript'];
+                return array_merge($all, ['javascript']);
             default:
-                return [];
+                return $all;
         }
```

Comment:
I would rather assign to a variable and merge `$all` afterwords instead of duplicating that logic for each case.